helm-chart/templates/11-ingress.yaml

```diff
@@ -33,7 +33,14 @@ spec:
                   number: 80
             path: /()(.*)
             pathType: Prefix
-  tls: {{ .Values.tap.ingress.tls | toYaml }}
+  tls:
+  {{- range .Values.tap.ingress.tls }}
+    - hosts:
+      {{- range .hosts }}
+        - {{ . | quote }}
+      {{- end }}
+      secretName: {{ .secretName }}
+  {{- end }}
```

Comment:
@batazor could you please elaborate the need of replacing `toYaml` helper with `range` operator?

Reply:
When trying to specify TLS for my ingress I encountered a problem converting JSON to YAML

```
tls:
  - secretName: kubeshark-ingress-tls
    hosts:
      - kubeshark.mysite.best
```

Reply:
@batazor OK, I understood. Thanks a lot!